### PR TITLE
demo: collapse custom demo supports dark mode

### PR DESCRIPTION
**Pull request by JarvisArt**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

image
image

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | collapse custom demo supports dark mode |
| 🇨🇳 Chinese | collapse custom demo 支持暗黑模式 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/collapse/demo/custom.tsx [hunk 1]
```diff
@@ -10,23 +10,34 @@ const text = `
...
+    borderRadius: token.borderRadiusLG,
+    border: 'none',
+  };
+
+  return (
+    <Collapse
+      bordered={false}
+      defaultActiveKey={['1']}
+      expandIcon={({ isActive }) => <CaretRightOutlined rotate={isActive ? 90 : 0} />}
+      className="site-collapse-custom-collapse"
```
Comment (afc163) [inline components/collapse/demo/custom.tsx:28]: 这个 site-collapse-custom-collapse 不需要了吧

Comment (JarvisArt) [reply components/collapse/demo/custom.tsx]: 没注意到这个className，我更一下

File: components/collapse/demo/custom.tsx [hunk 2]
```diff
@@ -10,23 +10,33 @@ const text = `
...
+  const panelStyle = {
+    marginBottom: 24,
+    background: token.colorFillAlter,
+    borderRadius: token.borderRadiusLG,
+    border: 'none',
+  };
+
+  return (
+    <Collapse
+      bordered={false}
```
Comment (afc163) [inline components/collapse/demo/custom.tsx:25]: 可以覆盖一下 Collapse 的 background 为白色，现在这个灰底有点奇怪。这里应该隔开，UI 上比较合理。

图片

Comment (afc163) [reply components/collapse/demo/custom.tsx:25]: 这个 demo从 3.x 之后就改坏了，一直延续至今。。。

https://2x.ant.design/components/collapse-cn/#components-collapse-demo-custom

**Review by afc163 — APPROVED** [review]

**Pull request merged**